### DISCUSSION TITLE
249C5647-7260-4687-B647-30EA66362D7F

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#109] [General]
# 249C5647-7260-4687-B647-30EA66362D7F

Gliese 486, also known as TOI-1827, is a nearby red dwarf star 26.35 light-years away in the constellation Virgo. In 2021, a hot superterran planet was discovered via both radial velocity and transit. The star and planet received the names Gar and Su at the end of the third NameExoWorlds campaign in 2023.

https://celestia.mobi/addon?item=249C5647-7260-4687-B647-30EA66362D7F

**Comment by EricNelson-sketch** [comment]
It's a good addon except the resolution is rather simple.
The number of subdivisions in the mesh is rather generic, and you can also tell b the poles.
It would look much more ideal with 2 more subdivisions.
This addon gets a C.

**Reply by EricNelson-sketch** [reply]
by the poles.
You can also tell easily that a higher resolution of the mesh would work when it comes to landing on the surface for a more accurate look.
